ext/exif/tests/bug77184/bug77184.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+Bug #74428 (Unsigned rational numbers are written out as signed rationals)
+--SKIPIF--
+<?php if (!extension_loaded('exif')) print 'skip exif extension not available';?>
+--INI--
+output_handler=
+zlib.output_compression=0
```

Comment:
I realize these are also present in some other exit test cases -- does someone know why? They don't look immediately exif related to me.

Reply:
I was just cargo-culting those into the test since I assumed they were important; I'm unfamiliar with the PHP testing ecosystem.

Reply:
@nikic I'm pretty sure they are a leftover from a copy-paste from another test. I guess they would origin from some test that was printing binary data or similar to compare it with something else. But they should most likely be removed, but thats something we can do in a separate commit after this PR

Reply:
Yeah, I agree that we should handle that separately.